autogpt_platform/Makefile

```diff
@@ -0,0 +1,19 @@
+.PHONY: start-core stop-core logs-core format lint
+
+# Run just Supabase + Redis + RabbitMQ
+start-core:
+	docker compose up -d kong auth db redis rabbitmq studio
```

Comment:
Use the profile deps so we don't have to update this when the dockerfile changes

Reply:
changed to using deps though its not a profile